### PR TITLE
Add another BEM reference

### PR DESCRIPTION
**Pull request by balvig**
I find this documentation to be a really clear and well-written reference for using BEM. 
Thanks to @sikachu for finding it!

One gotcha is that it endorses `block__element_modifier` rather than `block__element--modifier` which we use, so just watch out for that 😓

### CODE REVIEW

**Comment by brunobrgs** [comment]
Maybe add this `modifier` difference with the link, otherwise people can read and don't know that it is different from what we use.

**Comment by rikarumi** [comment]
I agree on @brunobrgs's point. I think we need to add that gotcha for clearer reference to what we use

**Comment by balvig** [comment]
@brunobrgs @rikarumi any suggestions on how to do that? (feel free to use suggestion feature etc)

File: scss/bem/README.md [hunk 1]
```diff
@@ -111,6 +111,7 @@ We try to have as much as possible of:
 
 ## Useful links
 
+- https://en.bem.info/methodology/quick-start/
```
Comment (brunobrgs) [inline scss/bem/README.md]: Something like: 
```suggestion
- https://en.bem.info/methodology/quick-start Gotcha: ignore `block__element_modifier` and use `block__element--modifier` instead
```

**Pull request merged**